webui/react/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "engines": {
     "node": ">=16.13 <17",
-    "npm": ">=8.0.0"
+    "npm": ">=9.4.2"
```

Comment:
**question:** is there a way to have multiple npm version ranges? Specifically something like:
```
"npm": ">=8.0.0 <9.0.0, >=9.4.2"
```
If we can do something like that, then we can skip the `use-npm-v9` step and just use the current npm version that comes with `16.14.2` for now and wait for the next node/npm upgrade so we don't have run an extra step to just update `npm`.

Reply:
i think we can do something like `"npm": ">=8.0.0  <= 9.4.2"`

Reply:
[Doc](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#dependencies) says we can do `"npm": ">=8.0.0 <9.0.0 || >=9.4.2"`